hpi-swt2/lecture-portal
# Fixed continuous deployment.

# Resolves #101

This PR fixes continuous deployment from dev.

Acceptance Criteria:
- [ ] deployment to dev triggers a deploy to [http://hpi-lectureportal-dev.herokuapp.com/](http://hpi-lectureportal-dev.herokuapp.com/)
- [ ] if docker build errors, Travis should also error


Testing steps:
- see last travis builds on this branch. They include examples of erroring and working deployment

Steps needed for migration:
- Run rails db:migrate. I modified it.

---

Definition Of Done:
- [ ] [ER-diagram](https://www.lucidchart.com/invitations/accept/66a4c292-9531-4468-ba60-2a99395ffce4) updated --> no changes
